apps/builder/app/shared/copy-paste/plugin-html.ts

```diff
@@ -1,14 +1,20 @@
 import { generateFragmentFromHtml } from "../html";
 import { insertWebstudioFragmentAt } from "../instance-utils";
+import { generateFragmentFromTailwind } from "../tailwind/tailwind";
 import { denormalizeSrcProps } from "./asset-upload";
 import type { Plugin } from "./init-copy-paste";
 
+const inceptionMark = `<!-- Generated with Inception by Webstudio -->`;
```

Comment:
I would use something like our version identifier for paste, might be helpful later on to have some version in it: aka `@webstudio/instance/v0.1` but `@webstudio/tailwind/v0.1`

Reply:
This way the intent of the comment is also clear and it has some structured data in it instead of a natural language string